pkg/manager/member/tikv_scaler.go

```diff
@@ -80,6 +80,11 @@ func (tsd *tikvScaler) ScaleIn(tc *v1alpha1.TidbCluster, oldSet *apps.StatefulSe
 		return err
 	}
 
+	if controller.PodWebhookEnabled {
+		decreaseReplicas(newSet, oldSet)
```

Comment:
Can webhook control the process of one by one scaling in? If it can now, `decreaseReplicas` is not needed.

Reply:
Currently, the webhook didn't control the process running step by step. We could transfer this scale-in logic in webhook in the future.